ext/storage_sources/local_store/local_store.c

```diff
@@ -235,7 +235,7 @@ local_config_dup(LOCAL_STORAGE *local, WT_SESSION *session, WT_CONFIG_ITEM *v, c
         free(p);
         return (ret);
     }
-    (void)strncat(p, suffix, len);
```

Comment:
This length argument to `strncat` is telling it how many chars from `suffix` to copy across **at most**. So Coverity is complaining that `p` is `len` characters long and we're trying to append `len` characters on top of what is already there. Since `len` is actually calculated by using `strlen(suffix)`, I don't think there's any possible UB here since it'll hit the null character in `suffix` before it copies across `len` characters.

But if we're relying on the `strlen(suffix)` in order to not overflow, we should just use `strcat` since the length argument is redundant in that case.

Reply:
Yes, it's worth a try.  If it fails, we could always fold the suffix into the `snprintf`, that will make the `strstr` test a little trickier (and I don't think `strnstr` is portable).